dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -3,15 +3,22 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:MUXControlsTestApp"
     xmlns:controls="using:Microsoft.UI.Xaml.Controls">
-    
 
-    <x:Double x:Key="NumberPanelButtonWidth">35</x:Double>
-    
-    <DataTemplate x:Key="NumberPanelButtonTemplate">
-        <Button Content="{Binding}" MinWidth="{StaticResource NumberPanelButtonWidth}"
-                Background="White"
-                Foreground="Black"/>
-    </DataTemplate>
+    <x:Double x:Key="NumberPanelButtonWidth">32</x:Double>
+
+    <Style x:Key="NumberPanelButtonStyle" TargetType="Button">
+        <Setter Property="MinWidth" Value="{StaticResource NumberPanelButtonWidth}"/>
+        <Setter Property="MinHeight" Value="20"/>
+        <Setter Property="Background" Value="Transparent"/>
+    </Style>
+
+    <Style x:Key="NumberPanelCurrentPageIdentifierStyle" TargetType="Rectangle">
```

Comment:
Is there a reason to have this as its own Style instead of setting the properties directly on the Rectangle?

Reply:
There really isn't since there is only one element that is going to use it.. I removed the style and applied them directly